cli/main.c

```diff
@@ -1354,6 +1358,14 @@ static int fw_update(int argc, char **argv)
 	if (type < 0)
 		return type;
 
+	if (switchtec_boot_phase(cfg.dev) == SWITCHTEC_BOOT_PHASE_BL1) {
```

Comment:
I'd suggest we have helper functions like switchtec_in_bl1/bl2/main() or similar to deal with the boot phase check.

Reply:
I disagree. That just sounds like a lot of unnecessary boiler plate code that doesn't add any additional clarity.

Reply:
I'm not feeling very strong to have it. However, I see a check of this like with current code of one level indent consumes 72 character in a line. With a helper function like switchtec_in_bl1() we will have less worry about the 80-character issue. And with helper like switchtec_in_bl1_main(), which is the check for some MRPC/CMD, we will further reduce the code and improve the readability.

Reply:
The goal isn't to have the fewest characters in a line. If there were a number of cases where we had to do the compare that weren't fitting on the line then maybe, but currently we have none.